src/v2/Apps/Artist/artistRoutes.tsx

```diff
@@ -120,84 +120,42 @@ export const artistRoutes: RouteConfig[] = [
         prepare: () => {
           WorksForSaleRoute.preload()
         },
-        prepareVariables: (params, props) => {
+        prepareVariables: ({ artistID }, props) => {
           // FIXME: The initial render includes `location` in props, but subsequent
           // renders (such as tabbing back to this route in your browser) will not.
           const filterStateFromUrl = props.location ? props.location.query : {}
 
           const filterParams = {
             ...initialArtworkFilterState,
             ...paramsToCamelCase(filterStateFromUrl),
-            ...params,
           }
 
-          filterParams.hasFilter = Object.entries(filterParams).some(
-            ([k, v]: [keyof ArtworkFilters, any]) => {
-              return !isDefaultFilter(k, v)
-            }
-          )
+          // filterParams.hasFilter = Object.entries(filterParams).some(
```

Comment:
oh yea, this was used at some point to try to prevent a fetch from happening that didn't need to, but the corresponding fragment [is also commented out](https://github.com/artsy/force/blob/81736e97f9fa319797b6867a62cd024b43094900/src/v2/Apps/Artist/Routes/Works/index.tsx#L156)